backend/src/main/scala/bloop/tracing/BraveTracer.scala

```diff
@@ -78,6 +90,7 @@ object BraveTracer {
     "http://127.0.0.1:9411/api/v2/spans"
   )
   val debugTrace = Properties.propOrFalse("zipkin.trace.debug")
+  val verboseTrace = Properties.propOrFalse("zipkin.trace.verbose")
```

Comment:
?
```suggestion
  val isDebugEnabled = Properties.propOrFalse("zipkin.trace.debug")
  val isVerboseEnabled = Properties.propOrFalse("zipkin.trace.verbose")
```

Reply:
Sounds good to me. Should we use the `bloop` prefix instead of `zipkin`?

Reply:
Using a bloop prefix would make sense.